solutions/client-landing-zone/client-folder/standard/org-policies/exceptions/compute-restrict-load-balancer-creation-for-types.yaml

```diff
@@ -30,7 +30,7 @@
 apiVersion: resourcemanager.cnrm.cloud.google.com/v1beta1
 kind: ResourceManagerPolicy
 metadata:
-  name: compute-restrict-load-balancer-creation-for-types-except-standard-folder
```

Comment:
any particular reason why the word compute is removed. have you faced a limit for metadata.name ?

Reply:
Not technical reason, felt it was redundant I guess given that it's for compute but I can add it back for to be more verbose. There's another issue with the `depends-on` we stumbled on this morning which is preventing the rest of the client package coming up when the name is corrected. Will need to debug some more. Issue does not exist on the previous package version.

Reply:
another option so everything is sorted by policy name when running `kubectl get gcp -n policies`

`compute-restrict-load-balancer-creation-for-types-except-client-name-standard-folder # kpt-set: compute-restrict-load-balancer-creation-for-types-except-${client-name}-standard-folder`

Reply:
oh that's a good point. Keep them nice and organized.

Reply:
Updated name.